src/VscPreviewExtension/Microsoft.DocAsCode.Dfm.VscPreview/DocfxProcessor.cs

```diff
@@ -13,69 +14,133 @@ namespace Microsoft.DocAsCode.Dfm.VscPreview
 
     public class DocfxProcessor
     {
-        public static string DocfxProcess(string baseDir, string relativePath, string markdownContent)
+        public static string DocfxProcess(string markdownContent)
         {
-            if (String.IsNullOrEmpty(baseDir) || String.IsNullOrEmpty(relativePath))
+            string baseDir = Console.ReadLine();
```

Comment:
It will be next PR.

---
In reply to: [105342657](https://github.com/dotnet/docfx/pull/1397#discussion_r105342657) [](ancestors = 105342657)